lettuce/flows/decayingturbulence.py

```diff
@@ -18,254 +17,100 @@ def __init__(self, resolution, reynolds_number, mach_number, lattice, k0=20, ic_
             characteristic_length_lu=resolution, characteristic_length_pu=2*np.pi,
             characteristic_velocity_pu=None
         )
+        self.wavenumber = []
```

Comment:
this should probably be `self.wavenumbers`

Reply:
We have to differ between the wavenumber array for the spectrum of zero to max(wavenorms) and the wavenumberS-array of shape (res,res,res) (3d) / (res,res) (2D), which is just important for the generating of the velocity.

The first array (wavenumber) can be extracted for the spectrum

Reply:
but this attribute (`wavenumber`) is not used anywhere in the code. it remains an empty list for its whole life time.

on the other hand, there is an attribute `self.wavenumbers` that is not created in `__init__`